libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Generators/ActivityFactory.cs

```diff
@@ -71,7 +67,7 @@ public static Activity GenerateFromLG(object lgResult)
         /// <param name="template">(optional) inline template definition.</param>
         /// <param name="data">data to bind the template to.</param>
         /// <returns>activity.</returns>
-        public async Task<Activity> Generate(ITurnContext turnContext, string template, object data)
+        public static async Task<Activity> Generate(ITurnContext turnContext, string template, object data)
```

Comment:
By removing IActivityGenerator, we want to remove this method also. 

This ActivityFactor should accept a string and return an object.